lib/matplotlib/tests/test_patheffects.py

```diff
@@ -132,3 +134,61 @@ def test_collection():
                                                        linewidth=3)])
         text.set_bbox({'boxstyle': 'sawtooth', 'facecolor': 'none',
                        'edgecolor': 'blue'})
+
+
+@image_comparison(['tickedstroke'], remove_text=True, extensions=['png'])
+def test_tickedstroke():
+    ax1 = plt.subplot(221)
```

Comment:
```suggestion
    fig, (ax1, ax2, ax3) = plt.subplot(1, 3, figsize=(12, 4))
```

Reply:
Is 

`fig, (ax1, ax2, ax3) = plt.subplot(1, 3, figsize=(12, 4))`

a new syntax?  When I try it on my local repo (not rebased to master), I get this error

`E           TypeError: subplot() takes 1 or 3 positional arguments but 2 were given`

I get what it is trying to do, but don't see how it is supposed to work.

Reply:
Should be `subplots`, not `subplot`.

Reply:
Thanks -- good eye.